hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java

```diff
@@ -564,6 +556,12 @@ class BootstrapPropertiesTest {
             entry(CONFIG_VERSION, 10),
             entry(RECORDS_USE_CONSOLIDATED_FCQ, false));
 
+    @LoggingTarget
```

Comment:
Not sure why this got moved.  It was perfectly fine where it was.  But ok.

Reply:
Very strange, added it to [the follow up task](https://github.com/hashgraph/hedera-services/issues/7449) that will address the code style issues.